cyclops-ui/src/components/pages/TemplateStore/TemplateStore.tsx

```diff
@@ -211,22 +222,26 @@ const TemplateStore = () => {
                   <Spin />
                 ) : (
                   <FileSyncOutlined
-                      className={classNames(
-                      styles.statustemplate, 
-                      {
-                        [styles.success]: requestStatus[template.name] === "success",
-                        [styles.error]: requestStatus[template.name] === "error"
-                      }
-                    )}
+                    className={classNames(styles.statustemplate, {
+                      [styles.success]:
+                        requestStatus[template.name] === "success",
+                      [styles.error]: requestStatus[template.name] === "error",
+                    })}
```

Comment:
Can you format it like this?

```suggestion
                    className={classNames(styles.statustemplate, {
                      [styles.success]: requestStatus[template.name] === "success",
                      [styles.error]: requestStatus[template.name] === "error",
                    })}
```